### PR TITLE
:sparkles: ambassors section

### PR DESCRIPTION
**Pull request by atilasousa**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206412083694649

### CODE REVIEW

File: views/home/ambassors/index.tsx
Comment (KipandaJr) [inline views/home/ambassors/index.tsx:1]: see how we do this on `TopBar`, we need a `Slider` dont this behavior

**Review by KipandaJr — CHANGES_REQUESTED** [review]

**Review by MarioBatalha — CHANGES_REQUESTED** [review]

File: views/home/ambassors/index.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        slidesToScroll: 2,
+      },
+    },
+    {
+      breakpoint: 600,
+      settings: {
+        slidesToShow: 1,
+        slidesToScroll: 1,
+      },
+    },
```
Comment (KipandaJr) [inline views/home/ambassors/index.tsx:35]: On settings plz add the `infinite: true`

File: views/home/ambassors/index.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+          <Box px="1rem" py="1rem">
+            <TitleCard />
+          </Box>
+          {AMBASSORS.map((ambassor) => (
+            <Box key={v4()} py="1rem" px="1rem">
+              <Card {...ambassor} />
+            </Box>
+          ))}
+        </Slider>
+        <Controllers handleNext={handleNext} handlePrevious={handlePrevious} />
```
Comment (KipandaJr) [inline views/home/ambassors/index.tsx:87]: check well if this Controllers r working well cuz here i have a weird behavior

File: views/home/ambassors/index.tsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+import { AMBASSORS } from './ambassors.data';
+import Card from './card';
+import TitleCard from './card/title-card';
+import Controllers from './controllers';
+
+const Ambassors: FC = () => {
+  const sliderRef = useRef<Slider>(null);
+
+  const sliderResponsiviness = [
```
Comment (KipandaJr) [inline views/home/ambassors/index.tsx:14]: Put this on another file `ambassors.data.ts`

**Review by KipandaJr — CHANGES_REQUESTED** [review]

File: views/home/ambassors/controllers/index.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import { Box, Button } from '@interest-protocol/ui-kit';
+import { FC } from 'react';
+
+import { CircleArrowLeftSVG, CircleArrowRightSVG } from '@/svg';
+
+interface ControllersProps {
```
Comment (git-marcopitra) [inline views/home/ambassors/controllers/index.tsx:6]: move it to `.types` file

File: views/home/ambassors/index.tsx [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+    {
+      breakpoint: 1100,
+      settings: {
+        slidesToShow: 3,
+        slidesToScroll: 3,
```
Comment (git-marcopitra) [inline views/home/ambassors/index.tsx:19]: slidesToScroll must be always 1

**Review by git-marcopitra — CHANGES_REQUESTED** [review]

**Review by KipandaJr — CHANGES_REQUESTED** [review]
image
the padding on mobile card r weird, plz check

**Review by MarioBatalha — APPROVED** [review]

File: views/home/index.tsx [hunk 6]
```diff
@@ -5,6 +5,7 @@ import Layout from '@/components/layout';
 
 import AboutSection from './about';
 import Advisor from './advisor';
+import Ambassors from './ambassadors';
```
Comment (KipandaJr) [inline views/home/index.tsx:8]: Change `Ambassors` to `Ambassadors`

**Review by KipandaJr — CHANGES_REQUESTED** [review]

**Review by git-marcopitra — APPROVED** [review]

**Pull request merged**